src/kava.js

```diff
@@ -62,6 +65,7 @@ class Kava extends BasePlugin {
     super(name, player, config);
     this._rateHandler = new KavaRateHandler();
     this._model = new KavaModel();
+    this._adsKava = new KavaAds(this.eventManager, player, this, this._model, this._sendAnalytics.bind(this));
```

Comment:
if there's no `logger` in the kava ads then you can omit the `this`
Also if you manage the model internally then no need for that.
And this all depends if you can think of a way to call the _sendAnalytics with different model stores, cause today we have explicit getter for the general model in it https://github.com/kaltura/playkit-js-kava/blob/3d812946ce233c431a3062c7df3c64813d0f27be/src/kava.js#L206-L221